pkgs/tools/text/uwuify/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, rustPlatform, ... }:
+
+rustPlatform.buildRustPackage {
+  pname = "uwu";
+  version = "1.0.0";
```

Comment:
Judging from the project tags, the appropriate version is `0.1.0` https://github.com/Daniel-Liu-c0deb0t/uwu/tree/v0.1.0.